scripts/update-size.js

```diff
@@ -6,13 +6,16 @@ const sizeLimit = require('size-limit');
 const root = path.join(__dirname, '..');
 const readmePath = path.join(root, 'README.md');
 const readme = fs.readFileSync(readmePath, 'utf-8');
+const sizeConfig = require('../.size-limit');
 
-const pkg = require(path.join(root, 'package.json'));
-const file = path.join(root, pkg.browser[pkg.module]);
-
-sizeLimit(file, { ignore: ['react', 'prop-types'] }).then(bytes => {
+sizeLimit(sizeConfig[0].path, sizeConfig[0].ignore).then(bytes => {
   const weight = formatBytes(bytes.gzip);
 
+  if (weight > sizeConfig[0].limit) {
+    // eslint-disable-next-line no-console
+    console.log(`\n⚠️  Project is now larger than ${sizeConfig[0].limit}\n`);
```

Comment:
that's good! I would only make it a console.warn instead of console.log

Reply:
Updated to console.warn 👍